docs/docs/flags/stores.1.md

```diff
@@ -0,0 +1,94 @@
+---
+title: TRACEE-STORES
+section: 1
+header: Tracee Stores Flag Manual
+date: 2025/12
+...
+
+## NAME
+
+tracee **\-\-stores** - Configure data stores for DNS cache and process tree
+
+## SYNOPSIS
+
+tracee **\-\-stores** [dns.enabled|dns.max-entries=*size*|process.enabled|process.max-processes=*size*|process.max-threads=*size*|process.source=*source*|process.use-procfs] [**\-\-stores** ...]
+
+## DESCRIPTION
+
+The **\-\-stores** flag allows you to configure data stores for DNS cache and process tree functionality.
+
+### DNS Store Options
+
+- **dns.enabled**: Enable the DNS cache store. When enabled, Tracee will cache DNS query information for enrichment of network events.
+
+- **dns.max-entries**=*size*: Set the maximum number of DNS query trees to cache. Default is 5000. Further queries may be cached regardless once the limit is reached.
+
+### Process Store Options
+
+- **process.enabled**: Enable the process tree store. When enabled, Tracee will maintain a tree of processes and threads for enrichment of events.
+
+- **process.max-processes**=*size*: Set the maximum number of processes to cache in the process tree. Default is 10928. This is an LRU cache that will evict least recently accessed entries when full.
+
+- **process.max-threads**=*size*: Set the maximum number of threads to cache in the process tree. Default is 21856. This is an LRU cache that will evict least recently accessed entries when full.
+
+- **process.source**=*source*: Set the source for process tree enrichment. Valid values are:
+  - **none**: Process tree source is disabled (default).
```

Comment:
Can we remove this "none" option now that we added the `enabled` config?
I believe we should remove it and make signals the default